experimental/FTheoryTools/src/LiteratureTateModels/constructors.jl

```diff
@@ -0,0 +1,111 @@
+@doc raw"""
+    literature_tate_model(; arxiv_id::String="", equ_nr::String="", description::String="")
+
+Many Tate models have been created in the F-theory literature.
+A significant number of them have even been given specific
+names, for instance the "U(1)-restricted SU(5)-GUT model".
+This method has access to a JSON-database, from which it can
+look up such literature models. Provided that the model is in
+our database, this method constructs the model including some
+of the information available in the literature.
+
+(Eventually, we hope to provide *all* information that is
+available on such models in the literature.)
+
+```jldoctest
+julia> t = literature_tate_model(arxiv_id = "1109.3454", equ_nr = "3.5")
+Global Tate model over a not fully specified base -- SU(5)xU(1) restricted Tate model based on arxiv paper 1109.3454 (equ. 3.5)
+
+julia> v = toric_ambient_space(t)
+Normal, simplicial toric variety
+
+julia> a10,a21,a32,a43,a65,w,x,y,z = gens(cox_ring(v));
+
+julia> I = ideal([x,y,w]);
+
+julia> v2 = blow_up(v,I)
+Normal toric variety
+
+julia> cox_ring(v2)
+Multivariate Polynomial Ring in 10 variables a10, a21, a32, a43, ..., e over Rational Field graded by
+  a10 -> [0 0]
+  a21 -> [0 0]
+  a32 -> [0 0]
+  a43 -> [0 0]
+  a65 -> [0 0]
+  w -> [1 0]
+  x -> [1 2]
+  y -> [1 3]
+  z -> [0 1]
+  e -> [-1 0]
+```
+"""
+function literature_tate_model(; arxiv_id::String="", equ_nr::String="", description::String="")
+  
+  # try to find the file with the desired model
+  @req (arxiv_id != "" || equ_nr != "" || description != "") "No information provided -- cannot perform look-up"
+    
+  # read index file
+  file_index = JSON.parsefile(joinpath(@__DIR__, "index.json"))
+  
+  # create list of possible candidate files
+  candidate_files = Vector{String}()
+  for k in 1:length(file_index)
+    test = true
+    if arxiv_id != "" && file_index[k]["arxiv_id"] != arxiv_id
+      test = false
```

Comment:
Can't you do away with the `test` variable and instead just `continue` here?
```suggestion
      continue
```

Reply:
Yes, that is a good idea. Thank you!